### PR TITLE
libisoburn.yaml: Add openssf-options

### PR DESCRIPTION
**Pull request by justinvreeland**

### CODE REVIEW

**Comment by justinvreeland** [comment]
```
2024/10/31 11:37:16 ERRO failed to test package: unable to build guest: unable to generate image: installing apk packages: error getting package dependencies: solving "libisoburn" constraint: could not find package either named cmd:wish or that provides cmd:wish for libisoburn
```

Ideally this would fix it but melange doesn't generate the provides:

```
❯ git diff 
diff --git a/libisoburn.yaml b/libisoburn.yaml
index a84152aa8..cbe50cf34 100644
--- a/libisoburn.yaml
+++ b/libisoburn.yaml
@@ -6,6 +6,9 @@ package:
   description: Library to enable creation and expansion of ISO-9660 filesystems
   copyright:
     - license: GPL-2.0-or-later
+  dependencies:
+    runtime:
+      - tk # https://github.com/chainguard-dev/melange/issues/1612
 
 environment:
   contents:
```

Not too worried about hacking past it for now.

**Review by smoser — CHANGES_REQUESTED** [review]

File: libisoburn.yaml [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
   name: libisoburn
   version: 1.5.6
-  epoch: 0
+  epoch: 1
   description: Library to enable creation and expansion of ISO-9660 filesystems
   copyright:
     - license: GPL-2.0-or-later
+  dependencies:
+    runtime:
+      - tk
```
[the remark this answers is not in the document]
Comment (justinvreeland) [reply libisoburn.yaml:11]: Both libisoburn and xiorriso are depending on wish which is provided by tk.

```
❯ tar -Oxz .PKGINFO < ./packages/aarch64/libisoburn-1.5.6-r1.apk | grep wish
depend = cmd:wish
❯ tar -Oxz .PKGINFO < ./packages/aarch64/xorriso-1.5.6-r1.apk | grep wish
depend = cmd:wish
```

[the remark this answers is not in the document]
Comment (smoser) [reply libisoburn.yaml:11]: right. but they both have /usr/bin/xorriso. the 'make install' that xorriso package does causes this.

```
6a3c8843461e:/# apk search cmd:xorriso
libisoburn-1.5.6-r0
xorriso-1.5.6-r0

```

[the remark this answers is not in the document]
Comment (smoser) [reply libisoburn.yaml:11]: fix coming

**Comment by smoser** [comment]
https://github.com/wolfi-dev/os/pull/34649

**Pull request closed without merging**